dpnp/dpnp_algo/dpnp_elementwise_common.py

```diff
@@ -538,32 +536,11 @@ def _call_divide(src1, src2, dst, sycl_queue, depends=None):
     return ti._divide(src1, src2, dst, sycl_queue, depends)
 
 
-def _call_divide_inplace(lhs, rhs, sycl_queue, depends=None):
```

Comment:
Could you please check `dpnp/dpnp_array.py` and implement missing in-place operations for already supported elementwise functions? (like `__imod__`)

Reply:
added definition for `__imod__`, `__floordiv__`, `__ifloordiv__`, and `__rfloordiv__`.